src/_providers.tf

```diff
@@ -24,3 +28,9 @@ provider "azurerm" {
   client_secret   = var.azure_service_principal.data.client_secret
   subscription_id = var.azure_service_principal.data.subscription_id
 }
+
+provider "azuread" {
```

Comment:
@chrisghill will need to review I don’t k ow if we have this provider in XO yet.